tests/NuGet.Services.BasicSearchTests/Data/V2SearchFunctionalTests.ReturnsCorrectSchema.json

```diff
@@ -28,9 +28,9 @@
             "DownloadCount": 30,
             "Dependencies": [],
             "SupportedFrameworks": [],
-            "Hash": "e7xbxwBo7NKNx1ZeyVgAdYdjl2diR71A4koclGbgQfQyV+8/7AhXpuiOF9mOHkg1f6M7aRVMbMhwSS+TRJHvWg==",
+            "Hash": "41n8nZ04XIPHUlXtiqyejpW2Xpdj/bg5xlWDwtLRp+29c8OfG0MVeMXj1swzOueCmEMKM7xEIwfQSJtqTuMcWg==",
             "HashAlgorithm": "SHA512",
-            "PackageFileSize": 4255228,
+            "PackageFileSize": 4264822,
```

Comment:
I believe we removed these types of checks altogether from Gallery functional tests... should we do the same here?